### ISSUE TITLE
sqlparse format using strip_comments=True removes new line char

### ISSUE THREAD

**Issue by sandeep-pareek** [andialbrecht/sqlparse#484]
While stripping the comments, it removes the new line char as well.

Code: 
```
import sqlparse

sql_file = """Create table myTable
 (
     myId TINYINT NOT NULL, --my special comment
     myName VARCHAR2(100) NOT NULL
 )"""
print (sqlparse.format(sql_file, strip_comments=True).strip())
```
Output: 
```
Sandeeps-MacBook-Pro:sandeep$ python myPy.py 
Create table myTable
 (
     myId TINYINT NOT NULL, myName VARCHAR2(100) NOT NULL
 )
```
This is particularly problematic for those relying on new lines to process sql files.

**Comment by andialbrecht** [comment]
That's a nifty issue. IMO a line break after a (single-line) comment should only be removed if the same line doesn't also has a valid statement. Unfortunately sqlparse has currently no concept of "the current line".

**Comment by ellbosch** [comment]
Running into this issue as well—we also need newlines retained when stripping comments. Is there an update on this request?

**Comment by ellbosch** [comment]
Pinging back on this—thanks.

**Comment by andialbrecht** [comment]
argh... sorry! I wanted to have this issue resolved in 0.3.1, but it slipped through :/
Working on it right now.

**Comment by rswgnu** [comment]
Would be very helpful.  Also would be great if you provided a simple method to extract a table alias name from a top-level query that involves sub-queries in the 'from' clause, e.g. select * from (subquery-here) alias where <conditions>.  What to return the 'alias' name without much effort.
